src/ertai/creature.py

```diff
@@ -0,0 +1,89 @@
+"""
+Class for Creature
+"""
+
+from .card import Card
+from dataclasses import dataclass
+
+
+@dataclass(repr=False)
+class Creature(Card):
+    """
+    A class for basic creatures.
+
+    Initiate this creature by giving power, toughness of it.
+
+    Other basic information below of this Creature derived from Card:
+
+    title : title of this creature
+
+    cost : cost to cast this creature
+
+    tapped : tap information of this creature
+
+    Next is the paramertes belong to Creature class:
+
+    Parameters:
+        - power : power  of this creature
+        - toughness : Toughness value of this creature,
+                but this value can be modified in a fight or other condition
+        - base_toughness : Toughness value of this creature,
+                use to store the base toughness of this creature.
+        - is_alive : A bool value represent wether this creature is alive.
+    """
+
+    power: int = 0
+    toughness: int = 0
```

Comment:
```suggestion
    power: int
    toughness: int
```

This will break one of your tests but I think we need to always pass a toughness and power to a creature.